ocaml/typing/jkind_types.ml

```diff
@@ -289,43 +277,43 @@ module Sort = struct
     and var ppf v = fprintf ppf "{ contents = %a }" opt_t !v
   end
 
-  let for_function = Const Const.value
+  let for_function = Const Value
```

Comment:
These RHSs should probably all be `value` instead of reconstructing `Const Value` for each